libraries/AC_AttitudeControl/AC_PosControl.cpp

```diff
@@ -818,13 +818,13 @@ void AC_PosControl::update_xy_controller()
     run_xy_controller(dt);
 
     // update xy update time
-    _last_update_xy_ms = now;
+    _last_update_xy_us = now_us;
 }
 
 float AC_PosControl::time_since_last_xy_update() const
```

Comment:
Is it just me or should we really be doing `micros_since_last_update`?  Obviously not on this PR :-)